examples/getting-started-with-serverless-framework/README.md

```diff
@@ -0,0 +1,143 @@
+# AWS Lambda and Serverless Framework ⚡️ Bolt for JavaScript
+
+> Slack app example for deploying to AWS Lambda using the Serverless Framework with Bolt for JavaScript
+
+## Overview
+
+This is an example app that updates the [Getting Started ⚡️ Bolt for JavaScript app][bolt-app] to be deployed to [AWS Lambda][aws-lambda] using the [Serverless Framework][serverless-framework].
+
+Before you begin, you may want to follow our [Getting Started guide][bolt-guide] to learn how to build your first Slack app using the [Bolt for JavaScript framework][bolt-website].
+
+## Getting started
+
+1. [Set up AWS credentials](#1-set-up-aws-credentials)
+1. [Set up local project](#2-set-up-local-project)
+1. [Create a Slack app](#3-create-a-slack-app)
+1. [Deploy to AWS Lambda](#4-deploy-to-aws-lambda)
+1. [Update Slack app settings](#5-update-slack-app-settings)
+1. [Test your Slack app](#6-test-your-slack-app)
+1. [Run on local machine](#7-run-on-local-machine)
+
+## 1. Set up AWS credentials
+
+### Install AWS CLI
```

Comment:
Sure, I've condensed this into a single section. Since this is the `README.md`, I felt atomic steps would be appropriate but happy to merge it into one section. See commit 7b93ff435ad830c747605bf022e8526b40b91ace